src/platform_impl/apple/appkit/mod.rs

```diff
@@ -27,26 +27,24 @@ pub(crate) use self::monitor::{MonitorHandle, VideoModeHandle};
 pub(crate) use self::window::{Window, WindowId};
 pub(crate) use self::window_delegate::PlatformSpecificWindowAttributes;
 pub(crate) use crate::cursor::OnlyCursorImageSource as PlatformCustomCursorSource;
-use crate::event::DeviceId as RootDeviceId;
 pub(crate) use crate::icon::NoIcon as PlatformIcon;
 pub(crate) use crate::platform_impl::Fullscreen;
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct DeviceId;
 
 impl DeviceId {
+    #[cfg(test)]
     pub const fn dummy() -> Self {
```

Comment:
Still need to remove this and other `fn dummy()`.

Or do we use it for testing anywhere that I don't know about?

Reply:
We have some tests that check if things don't panic:
https://github.com/rust-windowing/winit/blob/7fbc2118b6b395cd8028f82a5cc8b1563cc86076/src/event.rs#L1166-L1173

But it will be removed eventually when we split the backends out.

Reply:
I don't see that much value in that test compared to the many extra functions that we need to make it work - but I don't have a strong preference either way here.

Reply:
I feel the same and would be in favor of removing it.
Unless I hear disagreement I'm gonna go ahead with that when the PR is ready.